### ISSUE TITLE
Allow automatic time stepping the circulates through t=0

### ISSUE THREAD

**Issue by rsmacleod** [SCIInstitute/map3d#34]
The idea here is that the user could hold down the arrow key and move endlessly, looping through the time course of the data.

**Comment by bryanworthen** [comment]
I imagine this would be a toggleable feature?

**Comment by bryanworthen** [comment]
Feature should be complete.  I intentionally did NOT force multi-surface synchronization on looping - a surf that reaches the end will loop back to the beginning even if other surfaces haven't reached the end yet.  

(Use -fl on the command line or select 'Frame Looping' from Frame Control menu).

**Comment by rsmacleod** [comment]
Hi Bryan,

We chatted about this last week and would actually propose a slightly different wrap around behavior.

Could you set things so that all surfaces are held at the last time frame until the longest one is done, and then wrap back to the first index in all surfaces?  This seems like it would keep things best aligned over multiple plays of a beat.  This the sequence would like the following for 2 cycles of a 3-surface scenario in which surface 1 has 6 time instants, surface 2 has 4 and surface 3 has 5. 

1 2 3 4 5 6 1 2 3 4 5 6 ...
1 2 3 4 4 4 1 2 3 4 4 4 ...
1 2 3 4 5 5 1 2 3 4 5 5 ...

Your scenario, if I understood correctly, would result instead in 

1 2 3 4 5 6 1 2 3 4 5 6 ...
1 2 3 4 1 2 3 4 1 2 3 4 ...
1 2 3 4 5 1 2 3 4 5 1 2 ...

and you can see how in the second cycle already, things get offset and we get a mess with no alignment at all across surfaces. 

Thanks (and happy belated birthday)!
Rob

**Comment by bryanworthen** [comment]
Yes, you are correct in that that is what it would currently do, and yes, I
can make it happen as you describe.

Bryan

**Comment by bryanworthen** [comment]
And thanks for the birthday wishes!
